libathemecore/version.c

```diff
@@ -1,82 +1,98 @@
-  "  w00t, Robin Burchell <viroteck@viroteck.net>",
-  "  xiphias, Michael Rodriguez <dkingston@dkingston.net>",
-  "  zparta, Jens Holmqvist <zparta@hispan.se>",
-  " ",
-  "Portions of Atheme's code were derived from Shrike, which was mostly developed by",
-  "the following developers in nick-alphabetical order:",
-  "  madragoran, Mike Campbell <madragoran@avendesora.net>",
-  "  nenolod, William Pitcock <nenolod@nenolod.net>",
-  "  rakaur, Eric Will <rakaur@malkier.net>",
-  "  rintaun, Matt Lanigan <rintaun@projectxero.net>",
-  "  sycobuny, Stephen Belcher <sycobuny@malkier.net>",
-  " ",
-  "Visit our website at http://atheme.github.io",
-  0,
+	" ",
+	"Atheme IRC Services --",
+
+	"Copyright (C) 2004 Shrike Developers (http://www.malkier.net/)",
+	"Copyright (C) 2005-2016 Atheme Project (http://atheme.org/)",
+	"Copyright (C) 2015-2018 Atheme Development Group (https://atheme.github.io/)",
+	" ",
+
+	"Permission to use, copy, modify, and/or distribute this software for any",
+	"purpose with or without fee is hereby granted, provided that the above",
+	"copyright notice and this permission notice appear in all copies.",
+	" ",
+
+	"THE SOFTWARE IS PROVIDED \"AS IS\" AND THE AUTHORS DISCLAIM ALL WARRANTIES",
+	"WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF",
+	"MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR",
+	"ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES",
+	"WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN",
+	"ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF",
+	"OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.",
+	" ",
+
+	"Portions of Atheme's code were derived from Shrike, which was mostly"
+	"developed by the following developers in nick-alphabetical order:",
+	" ",
+	"  madragoran, Mike Campbell <madragoran@avendesora.net>",
+	"  nenolod, William Pitcock <nenolod@nenolod.net>",
+	"  rakaur, Eric Will <rakaur@malkier.net>",
+	"  rintaun, Matt Lanigan <rintaun@projectxero.net>",
+	"  sycobuny, Stephen Belcher <sycobuny@malkier.net>",
+	" ",
+
+	"Currently Atheme's core group consists of the following developers, in",
+	"nick-alphabetical order:",
+	" ",
+	"  ilbelkyr, Janik Kleinhoff <janik@kleinhoff.de>",
+	"  mt, Max Teufel <max@teufelsnetz.com>",
+	"  siniStar, Austin Ellis <siniStar@IRC4Fun.net>",
+	" ",
+
+	"The following people have contributed blood, sweat and tears to Atheme:",
+	" ",
+	"  elly, Elly <elly@leptoquark.net>",
+	"  jdhore, JD Horelick <jdhore1@gmail.com>",
+	"  jilles, Jilles Tjoelker <jilles@stack.nl>",
+	"  kaniini, William Pitcock <nenolod@nenolod.net>",
+	"  spb, Stephen Bennett <spb@exherbo.org>",
+	" ",
+
+	"The following additional people have contributed to Atheme:",
+	" ",
+	"  Adam, Adam <adam@sigterm.info>",
+	"  aji, Alex Iadicicco <alex@ajitek.net>",
+	"  alambert, Alex Lambert <alambert@quickfire.org>",
+	"  amdj, Aaron M. D. Jones <aaronmdjones@gmail.com>",
```

Comment:
I'll be honest, given the amount of work you've done for Atheme you should probably be listed at least in the group above, if not the core group (cc @maxteufel, @siniStar7boy about that one, though).

Reply:
I would agree on this completely and was actually thinking the exact same thing.